content/brukeropplevelse-og-datadeling/_index.en.md

```diff
@@ -0,0 +1,22 @@
+---
+title: User experience and data sharing
+ingress:
+
+# SEO
+description:
+keywords:
+
+banner:
+  image:
+    src: /illustrations/illustration-05.png
+    alt: Illustration of a person holding a mobile screen on his knee
+
+url: /en/user-experience-and-data-sharing
+
+navigation_link:
+  text: See what kind of data is available and how you can use it properly
+
+menu:
+  main:
```

Comment:
Nice catch! I removed all of them. But it seems like at least one page contents frontmatter should include a menu, otherwise site breaks, because the theme is using the menu without checking if it exists. So currently the main `_index.md` inside the content folder has this menu option, to avoid `baseOf.html` and `footer.html` breaking the site.